daprdocs/content/en/operations/configuration/configuration-overview.md

```diff
@@ -105,6 +105,19 @@ The following table lists the properties for metrics:
 |--------------|--------|-------------|
 | `enabled` | boolean | Whether metrics should to be enabled.
 
+To mitigate errors associated with [high cardinality metrics]({{< ref "metrics-overview.md#high-cardinality-metrics" >}}), you can set regular expressions for every metric exposed by the Dapr sidecar. For example:
```

Comment:
It's not errors that are mitigated, rather it's high memory usage and egress costs.